src/test/java/org/apache/sling/resourceresolver/impl/mapping/MapEntriesTest.java

```diff
@@ -2074,4 +2102,28 @@ public void run() {
             }
         }
     }
+
+    @Test
+    public void testIsValidAliasPath() throws Exception {
+        Method method = MapEntries.class.getDeclaredMethod("isValidAliasPath", String.class);
+        method.setAccessible(true);
+
+        // ignore system tree - path should not start with /jcr:system -
+        assertFalse((Boolean)method.invoke(mapEntries, "/jcr:system/node"));
+        //valid alias path
+        assertTrue((Boolean)method.invoke(mapEntries, "/parent"));
+        // notallowedparent is not valid configured alias path
+        assertFalse((Boolean)method.invoke(mapEntries, "/notallowedparent"));
+    }
+
+    @Test
+    public void testNullAliasPath() throws NoSuchMethodException, IllegalAccessException {
```

Comment:
This method does not fail if an exception is not thrown.  You probably want either `@Test(expected=...)` or to use the `ExpectedException` JUnit rule. I you remove the usage of reflection (see comment on previous method) you can use the first option.

Reply:
isValidAliasPath() is private method. That' why reflection is used in this test case.
The expected exception will be "InvocationTargetException", Inside catch assert statement has been written

`catch (InvocationTargetException e){
            assertEquals("Unexpected null path", e.getTargetException().getMessage());
        }`

Reply:
See my above comment - I suggested _not_ making this method private to ease testing.

Reply:
yeah understood, made required changes